packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -137,10 +137,43 @@ export interface ExpansionHub extends RevHub {
     ): Promise<PIDCoefficients>;
 
     // Servo
+    /**
+     * Set the interval between pulses (framePeriod) for a given servo
+     * @param servoChannel
+     * @param framePeriod time between the rising edge of each pulse
+     */
```

Comment:
We need to document the unit of framePeriod and include units in all of the parameter names.